.github/workflows/test.yaml

```diff
@@ -0,0 +1,27 @@
+on:
+    push:
+        branches:
+            - '*'
+jobs:
+    run:
+        runs-on: ${{ matrix.operating-system }}
+        strategy:
+            fail-fast: false
+            matrix:
+                operating-system: [ ubuntu-latest ]
+                php-versions: [ '7.4', '8.0' ]
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            -   name: Setup PHP, with composer and extensions
+                uses: shivammathur/setup-php@v2 #https://github.com/shivammathur/setup-php
+                with:
+                    php-version: ${{ matrix.php-versions }}
+                    extensions: mbstring
+
+            -   name: Install Composer dependencies
+                run: composer install --no-progress --prefer-dist --optimize-autoloader
+
+            -   name: Test
+                run: composer test
```

Comment:
would the GitHub publishing be part of the cd

Reply:
Packagist pulls new versions via a webhook